File: api/app/dominio/entidade/projeto.py

class projeto():
    id = ''
    cliente = ''
    nome = ''
    descricao = ''

    def __init__(self, id, cliente, nome, descricao):
        self.id = id,
        self.nome = nome,
        self.cliente = cliente,
        self.descricao = descricao


    def getValues(self):
        return {
            "id": self.id,
            "cliente": self.cliente,
            "nome": self.nome,
            "descricao": self.descricao,
        }

    def getResponseValues(self):
        cliente = self.cliente[0].getValues()

        return {
            "id": self.id[0],
            "cliente": self.cliente[0],
            "nome": self.nome[0],
            "descricao": self.descricao,
        }

File: pponto/src/app/entidades/projeto.ts
import { Cliente } from "./cliente";
import { Usuario } from "./usuario";

export class Projeto {
    id: number;
    cliente: Cliente;
    usuarios: Array<Usuario>
    coordenadores: Array<Usuario>
    nome: string;
    descricao: string;

    constructor() {
        this.id = 0;
        this.cliente = new Cliente();
        this.nome = '';
        this.descricao = '';
        this.usuarios = new Array<Usuario>();
        this.coordenadores = new Array<Usuario>();
    }
}